### ISSUE TITLE
Add scale_*_manual_grattan() wrappers

### ISSUE THREAD

**Issue by MattCowgill** [MattCowgill/grattantheme#7]
Add scale_colour_manual_grattan(), scale_fill_manual_grattan() and similar (or some variant) as a wrapper around scale_colour_manual(values = grattan_pal() ) etc.

**Comment by wfmackey** [comment]
@MattCowgill So the calling code would then be
`scale_colour_manual_grattan(3)`
rather than
`scale_colour_manual_grattan(values = grattan_pal(n = 3))`
?

**Comment by MattCowgill** [comment]
Yep, but the idea is that you could do _either_ `scale_colour_manual_grattan(n = 3)` _or_ `scale_colour_manual(values = grattan_pal(n = 3))`. And yep all the other arguments would continue to work, it'd just be a wrapper around scale_colour_manual. Maybe it's not worth doing...

**Comment by MattCowgill** [comment]
added this now as grattan_colour_manual()

doing `ggplot(data, aes(x, y)) + geom_whatever() + grattan_colour_manual(n = 2)`
is the same as doing
`ggplot(data, aes(x, y)) + geom_whatever() + scale_colour_manual(values = grattan_pal(n = 2))`
